csv-bulk-signature/src/main.ts

```diff
@@ -278,8 +247,8 @@ async function main() {
   // 4. generate signatures + report + zip
   await generateSignatures(template, contactsWithNewProps);
   await createStatusReport(contactsWithNewProps);
-  const zipFilePath = await zipUpFile();
-  await checkZipIsNotEmpty(zipFilePath);
+  await zipUpFile();
+  await checkZipIsNotEmpty(ZIP_FILE);
```

Comment:
allowed zip file name to be consistent across the board by creating it as a const